lib/templates/dlg/streams.genie

```diff
@@ -27,6 +27,10 @@ import * as S from '../state_manip';
 import ThingpediaLoader from '../load-thingpedia';
 
 notification_preamble : S.ContextInfo = {
+    ["reminder:", priority=1] : (ctx : ctx_notification_for_reminder) => ctx.currentFunction!.qualifiedName === '@org.thingpedia.builtin.thingengine.say' && 
+        ctx.state.current!.stmt.stream instanceof Ast.FunctionCallExpression && (ctx.state.current!.stmt.stream.name === 'ontimer' 
+        || ctx.state.current!.stmt.stream.name === 'attimer' || ctx.state.current!.stmt.stream.name === 'timer') ? ctx : null,
```

Comment:
You'll probably want to move this logic to a separate file in lib/templates/dialogue_acts